x-pack/plugins/lens/public/app_plugin/app.test.tsx

```diff
@@ -741,64 +815,59 @@ describe('Lens App', () => {
             },
           },
         });
-
-        expect(args.docStorage.save).toHaveBeenCalledWith({
-          id: '1234',
-          title: 'hello there2',
-          expression: 'kibana 3',
-          state: {
-            filters: [unpinned],
+        expect(services.attributeService.wrapAttributes).toHaveBeenCalledWith(
+          {
+            savedObjectId: defaultSavedObjectId,
+            title: 'hello there2',
+            expression: 'kibana 3',
+            state: {
+              filters: [unpinned],
+            },
           },
-        });
+          true,
+          { id: '5678', savedObjectId: defaultSavedObjectId }
+        );
       });
 
       it('checks for duplicate title before saving', async () => {
```

Comment:
What happens for duplicate title checks on by-value?

Reply:
Since duplicate titles are only a concern in the visualize library, and custom dashboard panel titles are an entirely separate system, there is no duplicate title checking when saving by value.